video/vdu.h

```diff
@@ -1,21 +1,114 @@
+#ifndef VDU_H
+#define VDU_H
+
 #include "agon.h"
 #include "cursor.h"
 #include "graphics.h"
 #include "vdu_audio.h"
 #include "vdu_sys.h"
 
+// Handle VDU commands
+//
+void VDUStreamProcessor::vdu(uint8_t c) {
```

Comment:
this function is identical to it's predecessor - it's just been shifted upwards in this file to aid readability/understandability of the code